### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by ruudvdent** [WAM2layers/WAM2layers#206] [Users of WAM2layers]
Hi all,

My name is Ruud van der Ent and I am one of the developers of the WAM2layers code. Here you're welcome to ask questions and exchange ideas, however, please note that my own time is limited, so I do hope that everyone from the WAM2layers community: developers, experienced users and beginners are going to engage here.

**Comment by Peter9192** [comment]
Hello,

I'm Peter, I'm a research software engineer at the Netherlands eScience Center, where we collaborate with researchers on many things software-related. I joined the WAM2layers developers last year and helped to rewrite the code for the upcoming version 3 release. I'm happy to help out with technical questions and guidance on best practices.

**Comment by YiyingWang-00** [comment]
Hello,
My name is Yiying Wang and I'm a Ph.D. student at Beijing Normal University. Recently I want to do research on tracking the moisture in China, using the WAM2layers. I hope we can communicate more together!

**Comment by Imme1992** [comment]
Hi all,
My name is Imme Benedict and I am affiliated to the Meteorology and Air Quality Group from Wageningen University as a researcher and lecturer. I have been working with the WAM2layers code for some years now, and am currently involved in improving the code to make it more user-friendly, and easy to use.

**Comment by chrisweij** [comment]
Hi all,

My name is Chris Weijenborg, I am also affiliated to Meteorology and Air Quality group of the Wageningen University. I am working on large-scale dynamic meteorology and I am interested in how (extra-tropical) cyclones are related to moisture transport and sources. I am also interested in improving wam2layers in applying e.g. the diagnostic along a track of a cyclone.

**Comment by XINGXING-ZHANG** [comment]
Hi all,
My name is Xingxing Zhang, I am a associate researcher from State Key Laboratory of Resources and Environmental Information System, Institute of Geographic Sciences and Natural Resources Research, Chinese Academy of Sciences. I have been engaged in hydrological and water resources research based on satellite remote sensing. I am very interested in the WAM2layers model and greatly appreciate the research conducted by your team. Using this model, I have also achieved some applied results, which have been published in the Journal of Hydrology (DOI：http://dx.doi.org/10.1016/j.jhydrol.2023.129562). Currently, I am interested in studying the impact of volcanic eruptions on global water vapor circulation using the WAM2layers model. I am also very willing to contribute my efforts to the improvement and enhancement of WAM2layers. If anyone is interested, please feel free to contact me for potential collaboration.

**Comment by P-Myers** [comment]
Hi everyone,

My name is Parker Myers. I am currently a master's student at the Eberswalde University for Sustainable Development in Brandenburg, Germany. I am currently working on my master's thesis exploring transboundary water dependencies, and hope to use the WAM2layers model to identify the precipitationshed for the Berlin-Brandenburg region. Additionally, I am curious to find other applications of the model's outputs, such as possibly calculating estimates of moisture's 'staying capacity' within a given region. I am very new to coding and large data management, and appreciate any support that is available here. I look forward to getting my model running soon, and getting in contact with the group, cheers!

**Reply by Peter9192** [reply]
Welcome! Good luck with your thesis and don't hesitate to seek help here :-)
